safe/gis/vector/summary_4_exposure_summary_table.py

```diff
@@ -44,7 +47,8 @@
 
 
 @profile
-def exposure_summary_table(aggregate_hazard, callback=None):
+def exposure_summary_table(
+        aggregate_hazard, exposure_summary=None, callback=None):
```

Comment:
:D 

Hmm, no. Since in this functionalities, I need to put the number of affected_something per exposure class. I think exposure summary table is the most suitable.